src/keyszer/output.py

```diff
@@ -127,8 +127,10 @@ def send_combo(self, combo):
             pressed_mod_keys.append(key)
 
         # normal key portion of the combo
+        sleep_ms(_THROTTLES['key_pre_delay_ms'])
         self.send_key_action(combo.key, PRESS)
         self.send_key_action(combo.key, RELEASE)
+        sleep_ms(_THROTTLES['key_post_delay_ms'])
```

Comment:
Now this is real since it's happening during output, not during config boot up.

Reply:
Yep. If that wasn't working I'd be very confused.